src/restconfig/src/test/java/org/geoserver/rest/resources/ResourceControllerTest.java

```diff
@@ -310,6 +311,7 @@ public void testDirectoryXML() throws Exception {
     }
 
     @Test
+    @Ignore
```

Comment:
Can you add a comment as to why this test should be ignored? 
```suggestion
    @Ignore("why is this ignored....")
```

Reply:
I think it wasn't intentional, met me check

Reply:
No, I missed that one. Was added by @aaime at 4bd66b2e222e464e03458892ca7f9737def3be00
I'll see what's going on

Reply:
Yep it was part of the work in progress, that branch was not meant to be committed as-is